### PR TITLE
[Core] Passing string to python instead of KRATOS_WARNING

### PR DESCRIPTION
**Pull request by loumalouomega**
Avoid potential silencing

### CODE REVIEW

**Comment by philbucher** [comment]
thx for fixing, but `KRATOS_WARNING` cannot be silenced, the reason has to sth else I would say
Did you try to make a test fail?

**Comment by loumalouomega** [comment]
> thx for fixing, but `KRATOS_WARNING` cannot be silenced, the reason has to sth else I would say
> Did you try to make a test fail?

I will

**Comment by loumalouomega** [comment]
> > thx for fixing, but `KRATOS_WARNING` cannot be silenced, the reason has to sth else I would say
> > Did you try to make a test fail?
> 
> I will

But when i implemented it worked... (I think is something of the CI)

File: kratos/processes/from_json_check_result_process.cpp [hunk 1]
```diff
@@ -272,19 +276,22 @@ void FromJSONCheckResultProcess::FailMessage(
...
+            ss << "Error checking for variable " << rVariableName << " in GP " << GPIndex << ", results:\n" << rEntityType << " " << EntityId << " " << std::setprecision(mRelevantDigits) << ValueEntity << "!=" << std::setprecision(mRelevantDigits) << ValueJSON << "; rel_tol=" << mRelativeTolerance << ", abs_tol=" << mAbsoluteTolerance << std::endl;
         } else {
-            KRATOS_WARNING("FromJSONCheckResultProcess") << "Error checking for variable " << rVariableName << ", results:\n" << rEntityType << " " << EntityId << " " << std::setprecision(mRelevantDigits) << ValueEntity << "!=" << std::setprecision(mRelevantDigits) << ValueJSON << "; rel_tol=" << mRelativeTolerance << ", abs_tol=" << mAbsoluteTolerance << std::endl;
+            ss << "Error checking for variable " << rVariableName << ", results:\n" << rEntityType << " " << EntityId << " " << std::setprecision(mRelevantDigits) << ValueEntity << "!=" << std::setprecision(mRelevantDigits) << ValueJSON << "; rel_tol=" << mRelativeTolerance << ", abs_tol=" << mAbsoluteTolerance << std::endl;
         }
     }
+    #pragma omp critical
+    mErrorMessage += ss.str();
```
Comment (philbucher) [inline kratos/processes/from_json_check_result_process.cpp]:
```suggestion
    #pragma omp critical
    {
        mErrorMessage += ss.str();
    }
```

**Review by philbucher — COMMENTED** [review]
perfect (for some reason I thought you were doing this already)

minor comment then I approve

thx

**Review by philbucher — APPROVED** [review]

**Pull request merged**